setup.cfg

```diff
@@ -4,6 +4,9 @@
 # will need to generate wheels for each Python version that you support.
 universal=1
 
+[flake8]
+ignore = E226,E704,W503,W504,F401,F811
```

Comment:
I would rather fix the flake8 errors in tests rather than accepting to be more sloppy in the main library code

Reply:
Agree. 
E226,E704,W503,W504 are part of the default ignore list of flake8. I needed to add F401,F811 but when you specify an ignore list in setup.cfg, the default ignore list is dropped and you need to put those errors explicitely. 

While answering your comment, I found a way to avoid flake8 raising F401,F811 errors. I'll correct the PR.